mapbook-iOS/View controllers/PortalItemsListViewController.swift

```diff
@@ -53,7 +53,7 @@ class PortalItemsListViewController: UIViewController {
         //mapbook we created, using 'Offline mapbook' keyword while fetching
         //portal items.
         var keyword:String?
-        if let urlString = AppContext.shared.portal?.url?.absoluteString, urlString == "https://www.arcgis.com" {
+        if let urlString = AppContext.shared.portal?.url?.absoluteString, urlString == AppSettings.arcGISOnlineURLString {
```

Comment:
If you added the constant as a static member of `URL`, then you could rewrite this test as follows:

```suggestion
        if let url = AppContext.shared.portal?.url, url == .arcGISOnline {
```